README.md

```diff
@@ -1811,6 +1811,7 @@ func loadTemplate() (*template.Template, error) {
 			continue
 		}
 		h, err := ioutil.ReadAll(file)
+		defer file.Close()
```

Comment:
move to the front of file.IsDir()?

Reply:
Moved to the top, as suggested.
Not sure if it's really needed here, but let it be like that.